docs/guides/initialize-account.mdx

```diff
@@ -0,0 +1,191 @@
+    ```python
+    from qiskit_ibm_runtime import QiskitRuntimeService
+
+    service = QiskitRuntimeService(instance="CRN_B", name="account_A")
+    ```
+- The service tries to load the default saved account only if you don't provide a `token` or `name` when initializing a service.  However, if an `instance` is provided explicitly, the service tries to connect to that instance by using the default credentials. If there is any conflict, a warning is raised.
+
+    In the following example, the service tries to load the default credentials and use the instance `CRN_B` - even if there is a different instance specified in the default account:
+    ```python
+    from qiskit_ibm_runtime import QiskitRuntimeService
+
+    service = QiskitRuntimeService(instance="CRN_B")
+    ```
+
+<span id="auto-select"></span>
+### Automatically select the instance
+
+If you provide a token but **don't provide an instance CRN** when instantiating the service, `QiskitRuntimeService` authenticates to the account identified by the token and uses automatic instance selection to choose the most relevant instance for the requested task. If you have several instances available within your account, the service automatically chooses from the available instances, depending on the resource requested and these `QiskitRuntimeService` options (if set): `plans_preference`, `region`, `tags`.
+
+- **`plans_preference`**: The types of instance plans to prioritize. For example, if [`open`] is passed in, only Open Plan instances are available. Accepted values are `open` and `premium`. If `plans_preference` is not specified, free plans are prioritized over paid plans.
+- **`region`**: The instance region. Accepted values are `us-east` and `eu-de`.
+- **`tags`**: The instance tags. Accepts a list of tag name strings.
+
+<Admonition type="note">
+You can set up your saved credentials to automatically select an instance.
+</Admonition>
+
+#### Examples
+
+In this example, the service searches all instances that are available to the account to find ones that can access the specified backend:
+
+```python
+service = QiskitRuntimeService(token=token)
+```
+
+In this example, the service searches all instances that are available to the account in the EU region to find ones that can access the specified backend:
+
+```python
+service = QiskitRuntimeService(token=token, region="eu-de")
+```
+
+In this example, the service searches all instances that are available to the account and are tagged as `service` to find ones that can access the specified backend:
+
+```python
+service = QiskitRuntimeService(token=token, tags=['services'])
+```
+
+In this example, the service searches all premium and open instances that are available to the account to find ones that can access the specified backend. If a backend is in both a premium and an open plan instance, the premium plan instance is prioritized because `premium` is specified first in `plans_preference`.
```

Comment:
```suggestion
In this example, the service searches all premium and open instances that are available to the account to find ones that can access the specified backend. If a backend is in both a Premium and an Open Plan instance, the Premium Plan instance is prioritized because `premium` is specified first in `plans_preference`.
```